macOS/DuckDuckGo/Updates/UpdateController.swift

```diff
@@ -486,7 +486,10 @@ extension UpdateController: SPUUpdaterDelegate {
             return
         }
 
-        PixelKit.fire(DebugEvent(GeneralPixel.updaterAborted, error: error))
+        PixelKit.fire(DebugEvent(
+            GeneralPixel.updaterAborted(reason: error.localizedDescription.strippingFilePaths()),
```

Comment:
Just in case the error includes a path, we'll strip the path and everything after it. We only need the initial string content to be able to determine which error this maps to in the Sparkle codebase.